pyecharts/options/charts_options.py

```diff
@@ -293,3 +293,212 @@ def _convert_dict_to_string(self, title_style: dict, subtitle_style: dict):
             self.title_style += '{}="{}" '.format(k, v)
         for k, v in subtitle_style.items():
             self.subtitle_style += '{}="{}" '.format(k, v)
+
+
+class GraphicShapeOpts(BasicOpts):

```

Comment:
Hi all, is it the time to consider python attrs or python 3.7 dataclass. As you see here, a lot of repeating blocks for data structure. One of two will save us typing a lot, less boilerplates.